lib/web_console/session.rb

```diff
@@ -32,20 +32,27 @@ def find(id)
       # storage.
       def from(storage)
         if exc = storage[:__web_console_exception]
-          new(ExceptionMapper.new(exc))
+          exception_mappers = [ExceptionMapper.new(exc)]
```

Comment:
Can we have a class method on `ExceptionMapper` that follows the cause of an exception and builds the array itself? Maybe `ExceptionMapper.follow` like:

```ruby
def self.follow(exc)
  mappers = [new(exc)]

  while cause = (cause || exc).cause
    mappers << new(cause)
  end

  mappers
end
```